src/api/server.ts

```diff
@@ -258,6 +265,36 @@ export class WebServer {
 
         this.express.use(router);
     }
+
+    wrapJSONHandler(handler: ActionHandler, core: ApiNamespace): express.Handler {
+        const server = this.server;
+
+        function mergeRequestData(req: express.Request): {[key: string]: any} {
+            return Object.assign({}, req.query || {}, req.body || {}, req.params || {});
```

Comment:
We dont want req.params in here. In general API uses GET requests but since GET only supports a limited payload in the browser, it can fallback to a POST request. Params and data should be seperated and can be conflicting

Reply:
Ok will change